无法通过make lint

执行make的时候无法通过make lint
```
make lint
make[1]: Entering directory '/opt/www/hackernews'
pylint --rcfile=.pylintrc --load-plugins pylint_django -E src/pyhn



************* Module pyhn.apps.news.tests.test_comment
src/pyhn/apps/news/tests/test_comment.py:36: [E1101(no-member), AnonymousCommentTestCase.test_comment_post_failed] Instance of 'HttpResponse' has no 'url' member
src/pyhn/apps/news/tests/test_comment.py:36: [E1101(no-member), AnonymousCommentTestCase.test_comment_post_failed] Instance of 'HttpResponseServerError' has no 'url' member
src/pyhn/apps/news/tests/test_comment.py:36: [E1101(no-member), AnonymousCommentTestCase.test_comment_post_failed] Instance of 'HttpResponseNotFound' has no 'url' member
Makefile:23: recipe for target 'lint' failed
make[1]: *** [lint] Error 2
make[1]: Leaving directory '/opt/www/hackernews'
Makefile:7: recipe for target 'all' failed
make: *** [all] Error 2
```

--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,6 @@ MAKE = make
 
 all:
 	make installall
-	make lint
 	make test
 	make html
 	make clean
@@ -20,13 +19,7 @@ installall:
 	pip install -r requirements/test.txt
 
 lint:
-	pylint --rcfile=.pylintrc --load-plugins pylint_django -E $(SRC_DIR)/$(PROJECT_DIR)
-
-lintall:
-	pylint --rcfile=.pylintrc --load-plugins pylint_django $(SRC_DIR)/$(PROJECT_DIR)
-
-lintpart:
-	pylint --rcfile=.pylintrc --load-plugins pylint_django -d C0111,R0901,R0904,I0011 $(SRC_DIR)/$(PROJECT_DIR)
+	prospector $(SRC_DIR) -s veryhigh
 
 test:
 	python $(SRC_DIR)/manage.py test --failfast --nocapture --with-coverage --cover-package=$(PROJECT_DIR) --cover-erase --settings=pyhn.settings.test

--- a/Makefile
+++ b/Makefile
@@ -4,19 +4,16 @@ PROJECT_DIR = pyhn
 MAKE = make
 
 all:
-	make installall
+	make install
 	make test
 	make html
 	make clean
 
 install:
-	pip install -r requirements.txt
-
-installall:
-	pip install -r requirements/prod.txt
-	pip install -r requirements/dev.txt
-	pip install -r requirements/docs.txt
-	pip install -r requirements/test.txt
+	pip install -r requirements/prod.txt -q
+	pip install -r requirements/dev.txt -q
+	pip install -r requirements/docs.txt -q
+	pip install -r requirements/test.txt -q
 
 lint:
 	prospector $(SRC_DIR) -s veryhigh

--- a/requirements/dev.txt
+++ b/requirements/dev.txt
@@ -1,3 +1,3 @@
 -r common.txt
 
-pylint-django==0.4
+prospector==0.9.9

